Ethics.Rmd

```diff
@@ -0,0 +1,35 @@
+# Ethical Considerations
+
+In this section, we would like to point out that there may be some aspects to consider in terms of the consequences of how you manage or share your data. 
+
+This is a new policy, so it isn't quite clear yet what should be done in all circumstances. 
+
+Here we propose some of our thoughts about aspects we think might require extra consideration. We do not have all the answers and are open to any feedback or thoughts you have on this topic. Please use the feedback button at the bottom of the table of contents to give us your feedback.
+
+## Consent
+
+Make sure that the consent process for data collection allows for sharing the data in the way that you have planned. If you have consent to share the data, the data may still be consented to only be used for certain research purposes. Be careful that the data are shared in a way that reflects this. For example, it may be necessary to only share the data in a repository that requires people submit for access in which they describe what they plan to do with the data.
+
+## De-identification
+
+You don't technically have to de-identify data in order to share it. You need to go through the process with the Internal Review Board (IRB) and consent review to identify under what conditions the data CAN be shared. Those conditions may be very limiting but usually are not full "cannot be shared" restrictions.
+
+If you end up using de-identification methods, it is often recommended that you consult an an expert to ensure that the privacy of the research participants is protected as de-identification can be trickier than expected.
+
+## Raw vs Processed Data
+
+You may be wondering what data you should share - the totally raw data or the processed data?
+
+Perhaps one should think about what level of effort/funding/resources would be necessary to make data easy for someone to use the data from scratch, since slightly processed data while less convenient takes up less space/uses less resources. Likely it might be ideal to aim for a balance between making it easy for others to reuse the data, while being conscious of what is required to do so.
+
+## Data Quality
+
+What quality threshold is really needed to share data?
+What are the consequences of sharing poor-quality data?
+What are the consequences of not sharing data that is of reasonable quality that someone decided was poor quality?
+
+These questions probably require case-by-case considerations. If the data was used for a publication, then we would imagine it is of high enough quality to be shared. However if the data was produced in the process of your work (but not used for your ultimate findings) and it might not be as helpful to others, maybe it doesn't need to be shared, especially if the quality is not as high as you would like. If you change your data sharing plans based on data quality, you would need to discuss with the program officer. 
```

Comment:
```suggestion
These questions probably require case-by-case considerations. If the data was used for a publication, then we would imagine it is of high enough quality to be shared. However if the data was produced in the process of your work (but not used for your ultimate findings) and it might not be as helpful to others, maybe it doesn't need to be shared, especially if the quality is not as high as you would like. For example, you might collect imaging data that is blurry and unusable, or you might collect data where an instrument was malfunctioning. If you change your data sharing plans based on data quality, you would need to discuss with the program officer. 
```

What about some examples?

Reply:
I like the example idea - I wonder if the unusable example might turn people away from thinking about nuanced situations - so could we maybe just have your second example?

Reply:
Open to discussion about this! I added the malfunctioning instrument example.